package.json

```diff
@@ -56,7 +56,7 @@
   "binary": {
     "module_name": "bcrypt_lib",
     "module_path": "./lib/binding/",
-    "host": "https://github.com",
-    "remote_path": "/kelektiv/node.bcrypt.js/releases/download/v{version}/"
+    "host": "https://github.com/kelektiv/node.bcrypt.js/releases/download/",
+    "remote_path": "v{version}"
```

Comment:
Yeah, it makes the host line incorrect when thinking in the manner of a URI, but host here refers the bucket (since node-pre-gyp was designed for s3). The analogy in github's case would be this.

And [node-pre-gyp-github](https://github.com/bchr02/node-pre-gyp-github) needs the host and remote path in this version